tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -221,6 +221,43 @@ IncludeByKey.PASSTHROUGH {
 				Object.Widget.host-copier.1 {
 					stream_name 'SSP0 Playback'
 					pcm_id 0
+					num_input_audio_formats 6
+					num_output_audio_formats 1
+					Object.Base.input_audio_format [
+						{
+							in_bit_depth		8
+							in_valid_bit_depth	8
+							in_sample_type		$SAMPLE_TYPE_UNSIGNED_INTEGER
+						}
+						{
+							in_bit_depth		8
+							in_valid_bit_depth	8
+							in_sample_type		$SAMPLE_TYPE_A_LAW
+						}
+						{
+							in_bit_depth		8
+							in_valid_bit_depth	8
+							in_sample_type		$SAMPLE_TYPE_MU_LAW
+						}
+						{
+							in_bit_depth		16
+							in_valid_bit_depth	16
+						}
+						{
+							in_bit_depth		32
+							in_valid_bit_depth	24
```

Comment:
Is IPC4_TYPE_LSB_INTEGER the default for these (16/24)?
On host side we expose S24_LE, which is of LSB type.